inst/tutorials/chapter3/chapter3.Rmd

```diff
@@ -7,6 +7,8 @@ runtime: shiny_prerendered
 ```{r setup, include=FALSE}
 library(learnr)
 knitr::opts_chunk$set(echo = FALSE)
+library(EnvStats, quietly = TRUE, warn.conflicts = FALSE, verbose = FALSE) # For Pareto Distribution
```

Comment:
If you add new dependencies, you need to also add them to the `DESCRIPTION` file at the package root. like now this is

```
Imports: 
  bookdown,
  shiny,
  learnr,
  plot3D
```
and you would just add there.

Reply:
✅